src/header/Header.tsx

```diff
@@ -77,6 +98,16 @@ export function Header() {
     return spaces;
   };
 
+  const handleSnowButtonClick = () => {
+    if (isSnowOpen) {
+      setIsSnowOpen(false);
+      localStorage.setItem(SNOW_STORAGE_KEY, JSON.stringify(false));
```

Comment:
Can't it be set as boolean?

Reply:
localStorage/sessionStorage only likes string variables: "Storage only supports storing and retrieving strings. If you want to save other data types, you have to convert them to strings. For plain objects and arrays, you can use [JSON.stringify()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)." https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem